lib/DiscoveryStreamFeed.jsm

```diff
@@ -67,54 +67,31 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
     }
   }
 
-  async fetchLayout() {
-    const endpoint = this.config.layout_endpoint;
+  async fetchFromEndpoint(endpoint) {
     if (!endpoint) {
-      Cu.reportError("No endpoint configured for pocket, so could not fetch layout");
+      Cu.reportError("Tried to fetch endpoint but none was configured.");
```

Comment:
How do errors get reported? Is it obvious where the error is coming from, or would it be helpful to add a bit of context to the error, example "Tried to fetch Discovery Stream endpoint but none was configured."

Reply:
You do get a line number / stack trace, so that should be enough (although I could improve this by maybe adding a reference to the endpoint?

Reply:
Line numbers is fine, let's not worry about this too much then.